provider/provider.go

```diff
@@ -39,43 +34,26 @@ type System interface {
 	Clear() int
 	Close() error
 	Stat() (ReproviderStats, error)
+	SetKeyProvider(kp KeyChanFunc)
 	Provider
 	Reprovider
 }
 
 // KeyChanFunc is function streaming CIDs to pass to content routing
 type KeyChanFunc func(context.Context) (<-chan cid.Cid, error)
 
-// NewBlockstoreProvider returns key provider using bstore.AllKeysChan
-func NewBlockstoreProvider(bstore blocks.Blockstore) KeyChanFunc {
-	return func(ctx context.Context) (<-chan cid.Cid, error) {
-		return bstore.AllKeysChan(ctx)
-	}
-}
```

Comment:
Was this method moved elsewhere?

Reply:
it was removed because == `bstore.AllKeysChan` to all effects (i don't even understand the indirection, probably it was a intermediate step before function signatures were aligned. Anyways, the blockstore dependency in this module needs to go away.